packages/frontend/core/src/components/blocksuite/block-suite-editor/lit-adaper.tsx

```diff
@@ -194,7 +194,7 @@ const usePatchSpecs = (shared: boolean, mode: DocMode) => {
       () => (
         <>
           {portals.map(p => (
-            <Fragment key={p.id}>{p.portal}</Fragment>
+            <Fragment key={p.portal.key}>{p.portal}</Fragment>
```

Comment:
I think this change is unnecessary

Reply:
`portal.id` has been updated and cannot be used anymore

Reply:
portal.id is mainly used when querying here:

```ts
    if (name === 'disconnectedCallback') {
            setPortals(portals => portals.filter(p => p.id !== id));
            return;
          }
```

Reply:
The ID/key etc are very confusing. It seems unnecessary complicated. I have updated the code in the upstack #9354. Can you please take a look?